cli/cli/Services/SwaggerService.cs

```diff
@@ -419,10 +415,126 @@ void RewireSchema(OpenApiSchema schema)
 
 	public async Task<IEnumerable<OpenApiDocumentResult>> DownloadBeamableApis(BeamableApiFilter filter)
 	{
-		var selected = Apis.Where(filter.Accepts).ToList();
+		var selected = APIS.Where(filter.Accepts).ToList();
 		return await DownloadOpenApis(_downloader, selected).ToPromise(); //.ShowLoading("fetching swagger docs...");
 	}
 
+	public OpenApiDocument GetCombinedDocument(IEnumerable<OpenApiDocumentResult> apis)
+	{
+		var combinedDocument = new OpenApiDocument(apis.FirstOrDefault().Document);

```

Comment:
in general, do we need to be worried about duplicates? 
_if_ there was a duplicate, I think the `Add` function would throw an exception right? So I guess in that sense, we are covered?

Reply:
And how should we approach them here?

Reply:
Update - so I dig more into our `SwaggerService` code, and now it resolves duplicates based on the code that we already use for Unity/Unreal code generation. There are 3 approaches:
- if there is `OpenApiPathItem` duplicate, I just skip it, should I care?
- if there is a `OpenApiSchema` or `OpenApiSecuritySchema` duplicate I check if it is exactly the same as existing one, if yes then skip, if no it will try to add it and throw Exception
- if there is other type of duplicate it will try to add it and throw Exception, there is none ATM

Reply:
I think all that logic is fine :+1: